extensions/quartz/deployment/src/test/java/io/quarkus/quartz/test/GlobalMisfirePolicyTest.java

```diff
@@ -0,0 +1,60 @@
+package io.quarkus.quartz.test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import javax.inject.Inject;
+
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.quartz.SchedulerException;
+import org.quartz.SimpleTrigger;
+import org.quartz.Trigger;
+import org.quartz.TriggerKey;
+
+import io.quarkus.scheduler.Scheduled;
+import io.quarkus.scheduler.Scheduler;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class GlobalMisfirePolicyTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest test = new QuarkusUnitTest()
+            .withApplicationRoot((jar) -> jar
+                    .addClasses(Jobs.class)
+                    .addAsResource(new StringAsset(
+                            "quarkus.quartz.default-misfire-policy.cron=fire-now\n" +
+                                    "quarkus.quartz.default-misfire-policy.simple=smart-policy\n"),
```

Comment:
Thanks for the PR @mun711 

I am wondering if the configurations name could be:
```
quarkus.quartz.simple-trigger.default-misfire-policy
quarkus.quartz.cron-trigger.default-misfire-policy
```

This way, if there are any other defaults that apply per trigger then we'll add them onto the `quarkus.quartz.cron-trigger.` or `quarkus.quartz.simple-trigger.` namespaces.

What do you think? @mun711 @gsmet  @mkouba @famod ?

Reply:
not sure if we should expect any other global configurations per trigger type

Reply:
Even if we don't expect them now it may be reasonable not to close the door ;-). So +1 for @machi1990's proposal.

Reply:
changed to the following:
quarkus.quartz.cron-trigger.misfire-policy
quarkus.quartz.simple-trigger.misfire-policy=smart-policy

Removed "default" because it seems redundant
Added config properties per trigger but linked to the same config group - as I understand the different config namespaces should be  enough in case we want to extend in the future @mkouba , @machi1990

Reply:
Thanks @mun711